perf/src/index.tsx

```diff
@@ -1,16 +1,80 @@
+import { Provider, themes } from '@stardust-ui/react'
+import * as _ from 'lodash'
 import * as React from 'react'
 import * as ReactDOM from 'react-dom'
 
+import { ProfilerMeasure, ProfilerMeasureCycle } from '../types'
+
+// https://github.com/bvaughn/rfcs/blob/profiler/text/0000-profiler.md
+const Profiler = (React as any).unstable_Profiler
+
 const mountNode = document.querySelector('#root')
-const performanceExamples = require.context('docs/src/examples/', true, /.perf.tsx$/)
+const performanceExamplesContext = require.context('docs/src/examples/', true, /.perf.tsx$/)
+
+// Heads up!
+// We want to randomize examples to avoid any notable issues with always first example
+const performanceExampleNames: string[] = _.shuffle(performanceExamplesContext.keys())
+
+const asyncRender = (element: React.ReactElement<any>, container: Element) =>
```

Comment:
although `container` is absolutely correct, would suggest to rename it to `mountNode` - to better reflect the context it is used for. It make it possible to get semantics behind each of the args much easier

Reply:
Agree, I left it because `container` comes from typings